app-frontend/src/app/index.config.js

```diff
@@ -66,6 +66,14 @@ function config( // eslint-disable-line max-params
         };
     });
 
+    RollbarProvider.init({
+        accessToken: APP_CONFIG.rollbarClientToken,
+        captureUncaught: true,
+        payload: {
+            environment: process.env.NODE_ENV || 'production'

```

Comment:
https://rollbar.com/docs/notifier/rollbar.js/#global has an example -- `person` looks like where to put the information